src/features/dashboard/components/AppsTable/__tests__/apps-table.test.tsx

```diff
@@ -291,6 +303,18 @@ describe('Apps Table', () => {
     expect(filterDialog).not.toBeInTheDocument();
   });
 
+  it('should sort the table by app id in ascending order', async () => {
```

Comment:
@ahmed-deriv 
we are using mixed `should` and `Should` can we make it consistent at all the places?

Reply:
Fixed